Name = "resourceServiceErrors"
SID = 3

[[error]]
    Name = "ErrDatabase"
    StatusHTTP = 500
    Message = "Database error"
    Kind = 1

[[error]]
    Name = "ErrResourceNotExists"
    StatusHTTP = 404
    Message = "Resource is not exists"
    Kind = 2

[[error]]
    Name = "ErrResourceAlreadyExists"
    StatusHTTP = 409
    Message = "Resource already exists"
    Kind = 3

[[error]]
    Name = "ErrPermissionDenied"
    StatusHTTP = 403
    Message = "Permission denied"
    Kind = 4

[[error]]
    Name = "ErrTariffUnchanged"
    StatusHTTP = 400
    Message = "Tariff unchanged"
    Kind = 5

[[error]]
    Name = "ErrTariffNotFound"
    StatusHTTP = 404
    Message = "Tariff was not found"
    Kind = 6

[[error]]
    Name = "ErrResourceNotOwned"
    StatusHTTP = 403
    Message = "Can`t set access for resource which not owned by user"
    Kind = 7

[[error]]
    Name = "ErrDeleteOwnerAccess"
    StatusHTTP = 409
    Message = "Owner can`t delete has own access to resource"
    Kind = 8

[[error]]
    Name = "ErrAccessRecordNotExists"
    StatusHTTP = 404
    Message = "Access record for user not exists"
    Kind = 9

[[error]]
    Name = "ErrInternal"
    StatusHTTP = 500
    Message = "Internal error"
    Kind = 10

[[error]]
    Name = "ErrValidation"
    StatusHTTP = 400
    Message = "Validation error"
    Kind = 11

[[error]]
    Name = "ErrServiceNotExternal"
    StatusHTTP = 400
    Message = "Service is not external"
    Kind = 12

[[error]]
    Name = "ErrTCPPortNotFound"
    StatusHTTP = 400
    Message = "TCP Port was not found in service"
    Kind = 13

[[error]]
    Name = "ErrInsufficientStorage"
    StatusHTTP = 507
    Message = "Can`t allocate resources for volume"
    Kind = 14
